airflow/providers/amazon/aws/example_dags/example_appflow.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.providers.amazon.aws.operators.appflow import (
+    AppflowRecordsShortCircuit,
+    AppflowRunAfterOperator,
+    AppflowRunBeforeOperator,
+    AppflowRunDailyOperator,
+    AppflowRunFullOperator,
+    AppflowRunOperator,
+)
+
+SOURCE_NAME = "salesforce"
+FLOW_NAME = "salesforce-campaign"
+
+with DAG(
+    "example_appflow",
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
+    catchup=False,
+    tags=["example"],
+) as dag:
+
+    # [START howto_appflow_run]
```

Comment:
We try to be consistent across the AWS package. That reminds me we should definitely work on writing a README file to explain all this. Here are the 3 variants we are using depending on the use case:
- howto_operator_
- howto_sensor_
- howto_transfer_

```suggestion
    # [START howto_operator_appflow_run]
```

Reply:
This comment is valid for all the `howto_`

Reply:
To be fair, we just finished cleaning that up.  But yes, we should definitely push a doc containing all that standardizing.

Reply:
Done!